src/fklearn/preprocessing/schema.py

```diff
@@ -0,0 +1,115 @@
+import functools
+import inspect
+import pandas as pd
+import toolz
+
+from typing import Any, Callable, Dict, List, Optional, Union
+from fklearn.types import LearnerLogType, LearnerReturnType
+
+
+@toolz.curry
+def feature_duplicator(df: pd.DataFrame,
+                       columns_to_duplicate: Optional[List[str]] = None,
+                       columns_mapping: Optional[Dict[str, str]] = None,
```

Comment:
What's the idea for this parameter? 
Is it a way for the user to have a more flexible mapping than prefix/suffix?
I couldn't find where this is used (or tested)

Reply:
yes, that's the idea
hm... i'll add them